<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="PowerButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
	
				   border-radius:10px;
				   
                   
	
	border-image: url(:/pictures/po1.png);
	
                   }

                   QPushButton:checked{
                   
	border-image: url(:/pictures/po3.png);
	
	
				
	
	
                   }
                   
                   QPushButton:hover{
		
	background-color: rgb(0, 85, 255,0);

                   }

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addTmp_Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
				   border-radius:10px;
	
	border-image: url(:/pictures/add.png);
                                                                
                   }

                   QPushButton:pressed{
                   
                   }
                   
                   QPushButton:hover{
background-color: rgb(0, 85, 255);
                   
	border-image: url(:/pictures/add2.png);
                   }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="reduceTmp_Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
				   border-radius:10px;
	
	
	border-image: url(:/pictures/reduce.png);
                                                                
                   }

                   QPushButton:pressed{
                   
                   }
                   
                   QPushButton:hover{
                   
	
	background-color: rgb(0, 85, 255);
	border-image: url(:/pictures/reduce2.png);
                   }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="targetTmp_valuelabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 48pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>25</string>
   </property>
  </widget>
  <widget class="QLabel" name="targetTmp_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;楷体&quot;;
color: rgb(247,255,0);</string>
   </property>
   <property name="text">
    <string>目标温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="Unit_label_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>℃</string>
   </property>
  </widget>
  <widget class="QLabel" name="realTmp_valuelabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 48pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>25.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="Unit_label_1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>℃</string>
   </property>
  </widget>
  <widget class="QLabel" name="realTmp_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(247,255,0);
font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>实际温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="wind_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
font: 9pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>风速</string>
   </property>
  </widget>
  <widget class="QPushButton" name="windClose_Button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
	color: rgb(255, 255, 255);
				   border-radius:10px;
				   background-color: rgb(85, 170, 255,100);
                                                                
                   }

                   QPushButton:pressed{
                   
	
                   }
                   
                   QPushButton:hover{
                  background-color: rgb(0, 85, 255);
                   }

</string>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lowSpeedButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
	color: rgb(255, 255, 255);
				   border-radius:10px;
				   background-color: rgb(85, 170, 255,100);
                                                                
                   }

                   QPushButton:pressed{
                   
	
                   }
                   
                   QPushButton:hover{
                  background-color: rgb(0, 85, 255);
                   }

</string>
   </property>
   <property name="text">
    <string>低</string>
   </property>
  </widget>
  <widget class="QPushButton" name="middleSpeed_Button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="100">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="100">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="100">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="100">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="100">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="100">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="100">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="100">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="100">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
	color: rgb(255, 255, 255);
				   border-radius:10px;
				   background-color: rgb(85, 170, 255,100);
                                                                
                   }

                   QPushButton:pressed{
                   
	
                   }
                   
                   QPushButton:hover{
                  background-color: rgb(0, 85, 255);
                   }

</string>
   </property>
   <property name="text">
    <string>中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="highSpeed_Button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
	color: rgb(255, 255, 255);
				   border-radius:10px;
				   background-color: rgb(85, 170, 255,100);
                                                                
                   }

                   QPushButton:pressed{
                   
	
                   }
                   
                   QPushButton:hover{
                  background-color: rgb(0, 85, 255);
                   }

</string>
   </property>
   <property name="text">
    <string>高</string>
   </property>
  </widget>
  <widget class="QLabel" name="fee_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;楷体&quot;;
color: rgb(247,255,0);</string>
   </property>
   <property name="text">
    <string>费用</string>
   </property>
  </widget>
  <widget class="QLabel" name="fee_valuelabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 48pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>10.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="Unit_label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
font: 9pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>元  风速等级</string>
   </property>
  </widget>
  <widget class="QLabel" name="work_label1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
font: 9pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>工作模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="work_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wind_label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
font: 9pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>530</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
font: 9pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>温度调节</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 9pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="currentfee_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>2.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
font: 9pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>元</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;楷体&quot;;
color: rgb(247,255,0);</string>
   </property>
   <property name="text">
    <string>当前费用</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
